### PR TITLE
Fix minor typos

### PR DESCRIPTION
**Pull request by tensorush**
Spotted a few small typos.

### CODE REVIEW

File: src/benchmark.zig [hunk 1]
```diff
@@ -234,7 +233,6 @@ const Benchmark = struct {
     batch_transfers: std.ArrayList(tb.Transfer),
     batch_start_ns: usize,
     transfers_sent: usize,
-    tranfer_index: usize,
```
Comment (tensorush) [inline src/benchmark.zig:243]: This is a mistyped duplicate of the [`transfer_index` field below](https://github.com/tigerbeetle/tigerbeetle/pull/1074/files#diff-50c3a64cbc2e1c22db2c57d14955e6739c52d2ce9013133d934e55e616f159bfR241).

**Review by eatonphil — DISMISSED** [review]

**Comment by eatonphil** [comment]
Thanks Jora! This is great!

**Comment by tensorush** [comment]
Looks like the `Benchmark` pipeline failed to find the `NOMAD_BENCHMARK_HTTP_AUTH` GitHub secret.

**Comment by eatonphil** [comment]
> Looks like the Benchmark pipeline failed to find the NOMAD_BENCHMARK_HTTP_AUTH GitHub secret.

No problem about that, this will only succeed if run by someone on the team due to github permissions.

**Comment by eatonphil** [comment]
Don't worry about keeping it up-to-date on main. It will dismiss my approve-and-run every time. :)

**Comment by eatonphil** [comment]
(We already use merge queues that validate any branch against latest main.)

**Comment by tensorush** [comment]
Right, thanks for telling me! Sorry for the fuss 😅

**Review by eatonphil — APPROVED** [review]

**Comment by eatonphil** [comment]
(Whoops, sorry, forgot to also approve it. That's why it wasn't merging on its own.)

**Pull request merged**